src/webview/columns/data-column.tsx

```diff
@@ -31,28 +31,36 @@ export class DataColumn implements ColumnContribution {
 
     protected renderGroups(range: BigIntMemoryRange, memory: Memory, options: TableRenderOptions): React.ReactNode {
         const groups = [];
-        let words = [];
-        for (let i = range.startAddress; i < range.endAddress; i++) {
-            words.push(this.renderWord(memory, options, i));
+        let words: React.ReactNode[] = [];
+        for (let address = range.startAddress; address < range.endAddress; address++) {
+            words.push(this.renderWord(memory, options, address));
             if (words.length % options.wordsPerGroup === 0) {
-                groups.push(<span className='byte-group' key={i.toString(16)}>{words}</span>);
+                this.applyEndianness(words, options);
```

Comment:
According to the issue

> It should be on `word` level of the display

But here it appears to be applied at the group level. @jreineckearm, could you clarify how you'd like things handled?

Reply:
@colin-grant-work , we had an offline discussion about that which I missed to reflect here or on the issue. Apologies for that, I'll update the issue.
Bottom line is that my initial thought was to add it on word level as we have this in other debuggers. But we realized later that the additional "group" level could cause confusion if you have endianess boundaries within groups (more than 1 word per group). Also, doing this on group level allows you a more use case specific endianess configuration. For example if requiring the display of a big endian buffer for a particular peripheral in an otherwise little endian word world.